### PR TITLE
Migrate to PyScaffold

### PR DESCRIPTION
**Pull request by FernandoDorado**
## Description
The migration to PyScaffold has been done, along with the installation of the dependencies you suggested (syft version https://github.com/OpenMined/SyferText/tree/aa91c82868b85a13002301359c4452cc418e272e/tests). It has been tested on Linux and Windows and everything seems to work fine.
I have done 17 tests for the spacy tokenizer as you said. For this, I have relied on the official spacy tests, and I have been able to do almost all of them. All tests have been passed. 
To execute them:
`pytest tests/tokenizer/test_spacy_tokenizer.py`
 

## Affected Dependencies
syft 

## How has this been tested?
- Running on Windows 10 and Linux (WSL2)

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [X] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [X] My changes are covered by tests

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def read(fname):
     packages=find_packages(),
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    install_requires=["tqdm==4.36.1", "mmh3==2.5.1", "requests==2.22.0", "syft==0.4.0"],
+    install_requires=["tqdm==4.36.1", "mmh3==2.5.1", "requests==2.22.0", "syft==0.5.0rc1"],
```
Comment (AlanAboudib) [inline setup.py:23]: This is the write version of Syft that should be installed with SyferText 

https://github.com/OpenMined/PySyft/tree/alan_syfertext_lm

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

**Comment by FernandoDorado** [comment]
I have updated the copyright and licence files to put the correct names.

**Comment by FernandoDorado** [comment]
I have done 17 tests for the spacy tokenizer as you said. For this, I have relied on the official spacy tests, and I have been able to do almost all of them. All tests have been passed. 
To execute them:

`pytest tests/tokenizer/test_spacy_tokenizer.py`

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    # The font size ("10pt", "11pt" or "12pt").
+    # "pointsize": "10pt",
+    # Additional stuff for the LaTeX preamble.
+    # "preamble": "",
+}
+
+# Grouping the document tree into LaTeX files. List of tuples
+# (source start file, target name, title, author, documentclass [howto/manual]).
+latex_documents = [
+    ("index", "user_guide.tex", "SyferText Documentation", "Fernando Dorado", "manual")
```
Comment (AlanAboudib) [inline docs/conf.py:248]: Your name is still here as author :D

**Review by AlanAboudib — CHANGES_REQUESTED** [review]
One more thing ;)

**Review by AlanAboudib — APPROVED** [review]

**Review by AlanAboudib — APPROVED** [review]

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    importlib-metadata; python_version<"3.8"
+    tqdm==4.36.1
+    mmh3==2.5.
+    requests==2.22.0
+    black>=20.8b1
+    pytest>=5.3.5
+    pytest-black>=0.3.8
+    jupyter>=1.0.0
+    papermill>=1.2.1
+    syft==0.3.0.post0.dev1485+g80bc24993.dirty
```
Comment (AlanAboudib) [inline setup.cfg:58]:
```suggestion
    syft
```

File: setup.cfg [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    click>=7.1.2
+    pytest-cov
+
+dependency_links = 
+    git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft-0.3.0.post0.dev1485+g80bc24993.dirty
```
Comment (AlanAboudib) [inline setup.cfg:63]:
```suggestion
    git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+python_requires = >=3.7
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    tqdm==4.36.1
+    mmh3==2.5.
```
Comment (AlanAboudib) [inline setup.cfg:51]:
```suggestion
```

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    requests==2.22.0
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
```

File: setup.cfg [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    black>=20.8b1
+    pytest>=5.3.5
```
Comment (AlanAboudib) [inline setup.cfg:54]:
```suggestion
```

File: setup.cfg [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    pytest-black>=0.3.8
+    jupyter>=1.0.0
```
Comment (AlanAboudib) [inline setup.cfg:56]:
```suggestion
```

File: setup.cfg [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+    papermill>=1.2.1
```
Comment (AlanAboudib) [inline setup.cfg:57]:
```suggestion
```

File: setup.cfg [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+    syft
+    click>=7.1.2
+    pytest-cov
```
Comment (AlanAboudib) [inline setup.cfg:60]:
```suggestion
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    black>=20.8b1
+    pytest-black>=0.3.8
+    syft
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
    'syft-syfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft-syfertext'
```

File: setup.cfg [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+    click>=7.1.2
+
+dependency_links = 
+    git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft
```
Comment (AlanAboudib) [inline setup.cfg:56]:
```suggestion
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    black>=20.8b1
+    pytest-black>=0.3.8
+    'syft-syfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft-syfertext'
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
    'syft_syfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft_syfertext'
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 13 above)
Comment (AlanAboudib) [inline setup.cfg]:
```suggestion
    'syftsyfer @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syftsyfer'
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    black>=20.8b1
+    pytest-black>=0.3.8
+    'syft_syfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft_syfertext'
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
    'syftsyfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syftsyfertext'
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    black>=20.8b1
+    pytest-black>=0.3.8
+    'syftsyfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syftsyfertext'
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
    syftsyfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syftsyfertext
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 16]
```diff
@@ -0,0 +1,132 @@
...
+
+# Add here dependencies of your project (line-separated), e.g. requests>=2.2,<3.0.
+# Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
+# new major versions. This works if the required packages follow Semantic Versioning.
+# For more information, check out https://semver.org/.
+install_requires =
+    importlib-metadata; python_version<"3.8"
+    black>=20.8b1
+    pytest-black>=0.3.8
+    syftsyfertext @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syftsyfertext
```
Comment (AlanAboudib) [inline setup.cfg:52]:
```suggestion
    syft @ git+https://github.com/OpenMined/PySyft.git@alan_syfertext_lm#egg=syft
```

**Review by AlanAboudib — CHANGES_REQUESTED** [review]

**Review by AlanAboudib — APPROVED** [review]

**Pull request closed without merging**